Explorer/Assets/DCL/Settings/ModuleControllers/EnvironmentDistanceSettingsController.cs

```diff
@@ -20,17 +21,23 @@ public EnvironmentDistanceSettingsController(SettingsSliderModuleView view, Land
 
             view.SliderView.Slider.onValueChanged.AddListener(SetEnvironmentDistanceSettings);
             SetEnvironmentDistanceSettings(view.SliderView.Slider.value);
+
+            landscapeData.OnDetailDistanceChanged += OnEnvironmentDistanceSettingsChangedFromOutside;
```

Comment:
Can landscapeData change when we switch to and from worlds?

What is the lifecycle of EnvironmentDistanceSettingsController?

Reply:
Anyone who changes the `landscapeData` scriptable object (that is how we're currently modifying the detail distance in out sustem), will make the slider control be updated. No matter from where this parameter is changed.

I linked this through an event in order to update the slider (among other cases) when we change the Quality preset dropdown.